check_api/src/main/java/com/google/errorprone/fixes/SuggestedFixes.java

```diff
@@ -1023,11 +1024,17 @@ public CharSequence getCharContent(boolean ignoreEncodingErrors) throws IOExcept
     Options originalOptions = Options.instance(javacTask.getContext());
     for (String key : originalOptions.keySet()) {
       String value = originalOptions.get(key);
-      if (key.equals("-Xplugin:") && value.startsWith("ErrorProne")) {
```

Comment:
I'd rather use the string names of the flags here. `com.sun.tools.javac.main.Optionn` is an unsupported internal API that has changed in the past, and the literal flag names are part of javac's supported API.

Reply:
Ack! Since you assigned the PR to yourself: want me to update this, or have you already applied those changes internally?